packages/ovee/package.json

```diff
@@ -18,20 +18,25 @@
     "/dist"
   ],
   "dependencies": {
+    "@vue/reactivity": "^3.2.6",
+    "@vue/runtime-core": "^3.2.6",
     "lit-html": "^1.1.2",
```

Comment:
investigate bump to lit 2.x

Reply:
I found some changes compared to `1.x` https://lit.dev/docs/releases/upgrade/#lit-html

but it's still in version `2.0.0-rc.5`

Reply:
okay, let's keep it on our radar. Resolving for now.